subscription_management/lambda_function.py

```diff
@@ -29,7 +29,7 @@
 def respond_error(err, origin, status=400):
     return {
         'statusCode': status,
-        'body': "{\"error\": \"%s\"}" % err,
+        'body': f"{{\"error\": \"{err}\"}}",
```

Comment:
`{{` gets a literal `{` in there; double check that it works

Reply:
```
>>> err = "Lorem ipsum"
>>> f"{{\"error\": \"{err}\"}}"
'{"error": "Lorem ipsum"}'

```